fixes nosetest errors

ltclm opened the pull request.

gjn commented on chsdi/models/vector/stopo.py:
```diff
@@ -498,10 +498,9 @@ class SwissboundariesKanton(Base, Vector):
 
 register('ch.swisstopo.swissboundaries3d-kanton-flaeche.fill', SwissboundariesKanton)
 
-
```
@ltclm Please install pre commit hook (as described in README.md file)

ltclm commented:
the hook was installed as described in the README.md
I fixed the errors but then i forgot to add the changed file again.
I have updated the README.md and added one more test to prevent this in future.
https://github.com/geoadmin/mf-chsdi3/pull/1946
